### ISSUE TITLE
The country code selector only works on Chrome desktop

### ISSUE THREAD

**Issue by karimmortabit** [firebase/firebaseui-web#150]
Hi,
I'm using the sign-in UI provided by firebaseui-web (Phone number only).
The country code selector only works on Chrome desktop, I tried Chrome mobile, Safari desktop & mobile, Firefox desktop with no success. Basically when I click nothing happens, the dropdown doesn't show all the countries.

![capture d ecran 2017-05-31 a 04 39 01](https://cloud.githubusercontent.com/assets/3072971/26616095/9babdd82-45bb-11e7-8303-6e617864cdbe.png)

Any idea of what it might be ?

**Comment by bojeil-google** [comment]
This must be unique to your setup. I just tested Chrome mobile, Windows Firefox and Mac OS X Safari and the country selector works as expected. Try it in the demo app: https://fir-ui-demo-84a6c.firebaseapp.com/
Can you provide exact code or a URL to debug your situation?

**Comment by karimmortabit** [comment]
Hello,
Thanks for your time. Here is a link to the app : http://brigades-affinity.com/
I get this error when looking at the console on Safari (while clicking on the country code selector ): 
`undefined is not an object (evaluating 'window.dialogPolyfill.registerDialog')`
<img width="1422" alt="capture d ecran 2017-05-31 a 21 41 21" src="https://cloud.githubusercontent.com/assets/3072971/26655415/3afca63a-464a-11e7-9e84-e1c0ba0664b1.png">
It may come from my configuration as suggested, I will continue looking at it and update.

**Comment by bojeil-google** [comment]
You seem to building your own JS bundle for FirebaseUI. Are you sure you including dialog-polyfill?

**Comment by karimmortabit** [comment]
I'm basically integrating this with React and using npm. What's the best way to include dialog-polyfill ?

**Comment by karimmortabit** [comment]
I guess there is no reasons to complicate things... Just had to put this a the top of your index.html

```
<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/dialog-polyfill/0.4.7/dialog-polyfill.js"></script>

<script>
  var dialog = document.querySelector('dialog');
  dialogPolyfill.registerDialog(dialog);
  // Now dialog acts like a native <dialog>.
  dialog.showModal();
</script>
```
Thanks for suggesting to add `dialog-polyfill`, wasn't aware of that.

**Comment by bojeil-google** [comment]
They also have an npm module too: https://github.com/GoogleChrome/dialog-polyfill 
You can install it and require it in your code.

**Comment by channing-google** [comment]
Hm, dialog-polyfill is supposed to be included by default. I'll investigate why it's not being picked up.

**Comment by karimmortabit** [comment]
@bojeil-google I tried the npm module first with ES6 `import` and it didn't work.

**Comment by bojeil-google** [comment]
Thanks for getting back on this. @channing-google will investigate this.

**Comment by RafaelPlantard** [comment]
@karimmortabit I'm facing the issue with import, how did you solve it?

**Comment by RafaelPlantard** [comment]
I solved it with: `window.dialogPolyfill = dialogPolyfill;`

**Comment by rodrigoreis22** [comment]
I'm having the same problem... it works on Chrome Desktop, it doesn't work on Firefox desktop. The country selector dropdown doesn't open.

**Comment by rodrigoreis22** [comment]
Here's my url for testing: http://dev.arena.im/loginframe (click on the bottom link, Continue with phone number)

**Comment by bojeil-google** [comment]
It's the same error:
TypeError: window.dialogPolyfill is undefined 
Can you solve by manually adding dialogPolyfill ?
I will investigate why this is getting dropped from the npm build.

**Comment by bojeil-google** [comment]
This should be fixed in 2.5.1
https://github.com/firebase/firebaseui-web/releases/tag/v2.5.1
